netci.groovy

```diff
@@ -123,9 +123,9 @@ def osShortName = ['Windows 10': 'win10',
 
 			def newBuildJob = job(Utilities.getFullJobName(project, newBuildJobName, isPR)) {
         		steps {
-            		batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" x86 && build.cmd /p:OSGroup=Windows_NT /p:ConfigurationGroup=${configurationGroup} /p:SkipTests=true")
+            		batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" x86 && build.cmd /p:OSGroup=Windows_NT /p:ConfigurationGroup=${configurationGroup} /p:SkipTests=true /p:Outerloop=true /p:WithoutCategories=IgnoreForCI")
             		// Package up the results.
-            		batchFile("C:\\Packer\\Packer.exe .\\bin\\build.pack .\\bin")
+            		batchFile("C:\\Packer\\Packer.exe .\\bin\\build.pack .\\ bin packages")
```

Comment:
The space here is intentional.  In this case, we are packing starting at the root of the repository, but only including things in the `bin` and `packages` folders (since we need both now).